<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sicels</class>
 <widget class="QWidget" name="Sicels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sicels</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </item>
   <item>
    <widget class="QPushButton" name="OK_button">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
